### PR TITLE
initial express application setup

### PR DESCRIPTION
**Pull request by CRasico**
Add Express Application with Sample Health Check Client for the Session Manager

- [x] Add Express JS Server
- [x] Add Session Manager Client
- [x] Wire Up Session Manger Client
- [ ] Add Unit Tests
- [x] Test Locally

### CODE REVIEW

**Review by nhleto — APPROVED** [review]
We should add `yarn run format` to our pipeline

File: packages/poker-sesh-bff/index.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import express from "express";
+
+const app = express();
+const port = 8080;
+
+app.get( "/", ( req, res ) => {
```
Comment (nhleto) [inline packages/poker-sesh-bff/index.ts:6]: Double quotes reeee

**Comment by CRasico** [comment]
Going to add some unit tests to the GRPC service before merging, pretty sick tho 😄

File: packages/poker-sesh-bff/test/infrastructure/grpc-session-repository.test.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	test('getHealth successfully', async () => {
+		const grpcSessionRepository = getGrpcSessionRepository();
+
+		const healthResponse = new HealthResponse();
+		healthResponse.setStatus(HealthResponse.HealthStatus.HEALTHY);
+		
+		mockHealthClient 
+			.setup(client => client.check(TypeMoq.It.isAny(), TypeMoq.It.isAny()))
+			.returns(() => healthResponse as any);
+
```
Comment (CRasico) [inline packages/poker-sesh-bff/test/infrastructure/grpc-session-repository.test.ts:33]: Reminder to self, need to figure out how to properly mock GRPC services....

**Comment by CRasico** [comment]
#30 Need to solve for this bad boy in the future. Right now this is just kind of shit.

File: packages/poker-sesh-bff/src/infrastructure/repository/grpc-session-repository.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { promisify } from 'util';
+import { ISessionRepository } from '../../domain/i-session-repository';
+import { IHealthClient } from '../../../../poker-sesh-session-manager/src/protocol-buffers/health_grpc_pb';
+import {
+  HealthRequest,
+  HealthResponse
+} from '../../../../poker-sesh-session-manager/src/protocol-buffers/health_pb';
```
Comment (nhleto) [inline packages/poker-sesh-bff/src/infrastructure/repository/grpc-session-repository.ts:7]: shoudldnt we be referencing the package and not the file path

**Pull request merged**